airbyte-webapp/src/scss/_variables.scss

```diff
@@ -13,3 +14,4 @@ $spacing-lg: 15px;
 $spacing-xl: 20px;
 $spacing-2xl: 40px;
 $spacing-page-bottom: 150px;
+$defaultBottomMargin: 150px;
```

Comment:
this var has been replaced with $spacing-page-bottom above. Looks like there is a conflict somewhere.